### PR TITLE
add py-pytest to master version

### PR DESCRIPTION
**Pull request by joequant**
the master has pytest which needs to be added to dependents

### CODE REVIEW

**Comment by vvolkl** [comment]
Title suggestion: "gaudi: add py-pytest to master version"

File: var/spack/repos/builtin/packages/gaudi/package.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ class Gaudi(CMakePackage):
     depends_on('range-v3')
     depends_on('root +python +root7 +ssl +tbb +threads')
     depends_on('zlib')
+    depends_on('py-pytest', when='@36.9:', type=('build', 'run'))
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/gaudi/package.py:63]:
```suggestion
    depends_on('py-pytest', when='@36.3:', type=('test'))
```

**Comment by vvolkl** [comment]
Thanks @joequant. Is py-nose still needed for the test?

**Comment by alalazo** [comment]
@vvolkl Thanks for reviewing. Feel free to ping me when this is ready to be merged.

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**